### ISSUE TITLE
Minifying multiple css files return [Object object]

### ISSUE THREAD

**Issue by shivekkhurana** [carrot/roots-css-pipeline#10]
Hello, 

I really like this roots.cx platform while learning it, the following problem showed up.
Following code is a part of app.coffee. Notice `minify: true` : 

```
module.exports =
  ignores: ['readme.md', '**/layout.*', '**/_*', '.gitignore', 'ship.*conf']

  extensions: [
    js_pipeline(files: 'assets/js/*.coffee'),
    css_pipeline({
        files: [
            'assets/css/normalize.css',
            'assets/css/skeleton.css',  
            'assets/css/master.styl'
        ],
        out: 'css/build.css', 
        minify: true
    })
  ]
```

When I do a ```$ roots watch```, the app compiles successfully but `build.css` has only the string ```[object Object]``` inside of it. If I remove ```minify:true```, this problem goes and entire css is available.

![screenshot from 2015-07-24 11 15 48](https://cloud.githubusercontent.com/assets/1925158/8868383/65654096-31f5-11e5-8c88-e0b93dbd206f.png)

Please help me with this. Thank you :)

**Comment by clhenrick** [comment]
I'm also having this same issue with using a single stylus file.

**Comment by SevereOverfl0w** [comment]
Seeing this here also. :+1:

**Comment by SevereOverfl0w** [comment]
https://github.com/carrot/roots-css-pipeline/blob/master/lib/index.coffee#L90 Looks to me like this shouldn't happen, as the contents of the file are being appended to a string.

This doesn't seem to happen in the test suite however. Looks like the cause is something else (other than multiple files), especially as @clhenrick isn't seeing it either.

---

I added `console.log(all_contents)` to line `#144` of the compiled node module, and got the css, with no sign of it being an array or object in disguise. But still saw [Object object]. I figured CleanCSS is now using a custom object on return, that we weren't expecting (part of the v3 upgrade perhaps?)

---

https://github.com/jakubpawlowicz/clean-css#how-to-use-clean-css-api clean-css api is not specified as how we're using it. This works, there's potentially a clean-css bug here, but it stems from improper use of the clean-css api.

I'll PR now.
